src/Css/Specificity/Specificity.php

```diff
@@ -80,8 +92,8 @@ public function getValues()
      */
     public static function fromSelector($selector)
     {
-        $pattern_a = "  \#";
-        $pattern_b = "  (\.[\w]+)                     # classes
+        $idSelectorsPattern = "  \#";
```

Comment:
``.foo\#bar`` is a single class selector for the class ``foo#bar``. But you would count it as a selector containing an id selector. See https://mathiasbynens.be/notes/css-escapes